### DISCUSSION TITLE
weight decay excluding bias & layernorm

### DISCUSSION THREAD

**Discussion by goattier** [boostcampaitech2/mrc-level2-nlp-12#61] [General]
https://github.com/boostcampaitech2/mrc-level2-nlp-12/blob/48b7136fc99f3ce9d4c5d7d59972050614744004/code/dpr.py#L197-L213

orginal bert implementation 과 동일

- https://github.com/google-research/bert/blob/master/optimization.py#L65
- https://discuss.pytorch.org/t/weight-decay-only-for-weights-of-nn-linear-and-nn-conv/114348/3
